docs/nightly/en/user-guide/manage-data/overview.md

```diff
@@ -0,0 +1,196 @@
+# Overview
+
+## Update data
+
+### Update data with same tags and time index
+
+Updates can be efficiently performed by inserting new data.
+By default, if rows of data have the same tags and time index, the old data will be replaced with the new data.
+For more information about column types, please refer to the [Data Model](../concepts/data-model.md).
+
+:::warning Note
+Excessive updates may negatively impact query performance, even though the performance of updates is the same as insertion.
+:::
+
+To update data, you can insert new data with the same tag and time index as the existing data.
+This updating mechanism is supported by all GreptimeDB protocols in the [Ingest Data documents](/user-guide/ingest-data/overview.md). The following example uses SQL.
+
+Assuming you have a table named `monitor` with the following schema.
+The `host` column represents the tag and the `ts` column represents the time index.
+
+```sql
+CREATE TABLE monitor (
+    host STRING,
+    ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP() TIME INDEX,
+    cpu FLOAT64 DEFAULT 0,
+    memory FLOAT64,
+    PRIMARY KEY(host)
+);
+```
+
+Insert a new row into the `monitor` table:
+
+```sql
+INSERT INTO monitor (host, ts, cpu, memory)
+VALUES ("127.0.0.1", "2024-07-11 20:00:00", 0.8, 0.1);
+```
+
+Check the data in the table:
+
+```sql
+SELECT * FROM monitor;
+```
+
+```sql
++-----------+---------------------+------+--------+
+| host      | ts                  | cpu  | memory |
++-----------+---------------------+------+--------+
+| 127.0.0.1 | 2024-07-11 20:00:00 | 0.8  | 0.1    |
++-----------+---------------------+------+--------+
+1 row in set (0.00 sec)
+```
+
+To update the data, you can use the same `host` and `ts` values as the existing data and set the new `cpu` value to `0.5`:
+
+```sql
+INSERT INTO monitor (host, ts, cpu, memory)
+-- The same tag `127.0.0.1` and the same time index 2024-07-11 20:00:00
+VALUES ("127.0.0.1", "2024-07-11 20:00:00", 0.5, 0.1);
+```
+
+The new data will be:
+
+```sql
+SELECT * FROM monitor;
+```
+
+```sql
++-----------+---------------------+------+--------+
+| host      | ts                  | cpu  | memory |
++-----------+---------------------+------+--------+
+| 127.0.0.1 | 2024-07-11 20:00:00 | 0.5  | 0.1    |
++-----------+---------------------+------+--------+
+1 row in set (0.01 sec)
+```
+
+Note that you **cannot omit** the other columns in the `INSERT INTO` statement if you only want to update one column.
```

Comment:
Since v0.9.0, we have provided a new option `merge_mode` when creating tables.

If choosing `last_none_null`, you don't have to present all the columns when updating.

https://docs.greptime.com/reference/sql/create#create-a-table-with-merge-mode

Reply:
Another note: the influxdb tables created automatically are both `append_only=true` and `merge_mode=last_none_null`, which is compatible with the influxdb protocol semantic.

Reply:
how about gRPC, OTLP, and all the other protocols which can create table automatically?